src/ElementProxy.js

```diff
@@ -12,26 +12,31 @@ import {NodeProxy} from './types'
 export class ElementProxy {
   _node: HTMLElement;
 
+  // Expose these so they can be stubbed for unit test
+  _emitMount: Function;
+  _emitUnmount: Function;
+
   constructor (node: HTMLElement) {
     this._node = node
   }
 
   emitMount (fn: Function): void {
-    emitMount(this._node, fn)
+    this._emitMount(this._node, fn)
   }
 
   emitUnmount (fn: Function): void {
-    emitUnmount(this._node, fn)
+    // Rely on prototype method so it can be stubbed for unit test
+    this._emitUnmount(this._node, fn)
```

Comment:
Would all uses of `ElementProxy` be required to provide implementation for `emitMount` and `emitUnmount`? If so, that seems overly burdensome given that it is implementation detail that is not likely to be changed except for unit test.

How would references to the injected implementations be maintained? Currently, I imagine it would be the same as this PR but injected via constructor rather than defaults `set` on the prototype. I may need another clue because my imagination is mostly producing what is here.